com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -456,14 +456,6 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             requiresDepthPrepass |= m_DepthPrimingMode == DepthPrimingMode.Forced;
 
             // If possible try to merge the opaque and skybox passes instead of splitting them when "Depth Texture" is required.
-#if ENABLE_VR && ENABLE_XR_MODULE
-            // This optimization is temporarily disabled on VR/XR because of a depth copy y-flip issue on DX11.
-            if (cameraData.xr.enabled)
-            {
-                m_CopyDepthPass.renderPassEvent = (!requiresDepthTexture && (applyPostProcessing || isSceneViewCamera || isGizmosEnabled)) ? RenderPassEvent.AfterRenderingTransparents : RenderPassEvent.AfterRenderingOpaques;
-            }
-            else
-#endif
             {
```

Comment:
This open and close brackets should be removed and the block unindented.

Reply:
Done in 
f7926b2